internal/proxy/task.go

```diff
@@ -857,8 +857,14 @@ func (dpt *dropPartitionTask) PreExecute(ctx context.Context) error {
 		return err
 	}
 
-	collID, _ := globalMetaCache.GetCollectionID(ctx, dpt.GetCollectionName())
-	partID, _ := globalMetaCache.GetPartitionID(ctx, dpt.GetCollectionName(), dpt.GetPartitionName())
+	collID, err := globalMetaCache.GetCollectionID(ctx, dpt.GetCollectionName())
+	if err != nil {
+		return err
+	}
+	partID, err := globalMetaCache.GetPartitionID(ctx, dpt.GetCollectionName(), dpt.GetPartitionName())
+	if err != nil {
+		return err
```

Comment:
We should make `drop partition` idempotent, but this will break the rule. Btw, why we didn't directly call `drop partition by name`?